_components/form-controls/09-combo-box.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Combo box
+parent: Form controls
+order: 09
+lead: A combo box helps users select an item from a large list of options.
+---
+
+{% include code/preview.html component="combo-box" %}
+{% include code/accordion.html component="combo-box" %}
+<div class="usa-accordion usa-accordion--bordered site-accordion-docs">
+  <button class="usa-button-unstyled usa-accordion__button"
+      aria-expanded="true" aria-controls="combo-box-docs">
+    Guidance
+  </button>
+  <div id="combo-box-docs" aria-hidden="false" class="usa-accordion__content site-component-usage">
+    <h4>When to use the combo box component</h4>
+    <ul class="usa-content-list">
+      <li><strong>More than 15 options.</strong> When there are more than 15 choices in a drop-down list it can be hard to navigate with scrolling only.</li>
+      <li><strong>Limited space.</strong> Use a combo box for presenting options when screen real estate is limited.</li>
+    </ul>
+    <h4>When to consider something else</h4>
+    <ul class="usa-content-list">
+      <li><strong>Limited choices.</strong> When the number of options is small you can continue to use a <code>select</code> or <code>radio</code> elements.</li>
+    </ul>
+    <h4>Usability guidance</h4>
+    <ul class="usa-content-list">
+      <li><strong>Use option strings familiar to users.</strong> The combo box filters by matching strings. Include option text that includes familiar strings or spellings (i.e. if using the combobox with a state list, include the postal abbreviation in the option text: <code>District of Columbia (DC)</code>).</li>
```

Comment:
@Ben-Judy-Flexion I added this piece of guidance based on conversations we around combo box filtering. Does it make sense to you?

Reply:
Yes! Brilliant.